tests/_helpers.py

```diff
@@ -3,11 +3,12 @@
 import pytest
 
 wrapped_libraries = ["numpy", "cupy", "torch", "dask.array"]
-all_libraries = wrapped_libraries + ["array_api_strict", "jax.numpy", "sparse"]
-
+all_libraries = wrapped_libraries + [
+    "array_api_strict", "jax.numpy", "ndonnx", "sparse"
```

Comment:
This too. Why are we testing here libraries which we don't wrap?

Reply:
Because `_helpers.py` is chock full of special cases and exceptions for them.